applications/ultrasound_postprocessing/ultra_post/filters/colormap.py

```
@@ -0,0 +1,107 @@
+    scale = cp.clip(scale, 0.0, 4.0)[..., None]
+    colored = colored * scale
+
+    blend = float(blend)
+    if blend < 1.0:
+        gray_rgb = cp.stack((y, y, y), axis=-1)
+        colored = blend * colored + (1.0 - blend) * gray_rgb
+
+    return cp.clip(colored, 0.0, 1.0).astype(cp.float32, copy=False)
+
+
+__all__ = ["color_map", "MapChoice"]
```

Ruff (0.14.13)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)